src/components/Profile/components/Identification/components/SocialMediaLinks/components/LinksDialog/index.jsx

```diff
@@ -60,9 +64,11 @@ const LinksDialog = ({ links, createSnackbar, onClose, setLinks }) => {
       console.log('Failed Updates: ', failedUpdates);
       createSnackbar('Failed Updating Link(s)', 'error');
     } else {
-      onClose();
       createSnackbar('Social Media Links Updated', 'success');
+      updateProfile();
+      //  setIsSubmitted(true);
```

Comment:
Can you remove this commented out code, so that it doesn't confuse future programmers?